client/src/components/ExpertiseChips.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { Chip, Typography, TextField, IconButton } from "@material-ui/core";
+import { Add } from "@material-ui/icons";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  chip: {
+    margin: "2px",
+  },
+}));
+
+function ExpertiseChips(props) {
+  const classes = useStyles();
+
+  const [chips, setChips] = useState(props.expertiseList);
+  const [expertise, setExpertise] = useState("");
+
+  useEffect(() => {
+    props.onStateChange(chips);
+  }, [chips]);
+
+  const handleAddChip = (event) => {
+    setChips((chips) => {
+      // If the industry is already chosen, we don't want a duplicate
+      if (chips.includes(expertise)) return [...expertise];
+      else return [...chips, expertise];
+    });
+    setExpertise("");
+  };
+
+  const handleRemoveChip = (chip) => {
+    setChips((chips) => {
+      const index = chips.indexOf(chip);
+      chips.splice(index, 1);
+      return [...chips];
```

Comment:
```suggestion
      return chips.filter((c) => return c !== chip)
```

i _think_ that should be a similar and shorter way to filter out that specific chip :)

Reply:
Much cleaner, thanks!